docs/source/packages/accounts-2fa.md

```diff
@@ -0,0 +1,140 @@
+
+> This package is meant to be used with `accounts-password`, so if you still didn't add `account-password` to your project, you'll need to add it too. In the future we want to enable the use of this app with other login methods, like `accounts-passwordless` or our oauth methods (Google, GitHub, etc...).
+
+<h3 id="activating-2fa">Activating 2FA</h3>
+
+The first step to use 2FA is to generate a QR code so the user can read it on an authenticator app and start to receiving codes.
+
+{% apibox "Accounts.generate2faActivationQrCode" "module":"accounts-base" %}
+
+Receives an `appName` which is the name of your app that will show up when the user scans the QR code. Also, a callback called with a QR code in SVG format on success or a single `Error` argument
+on failure. Both parameters are optional.
+
+On success, this function will also add an object to the logged user containing the QR secret:
+
+```js
+twoFactorAuthetication: {
+  secret: "***"
+}
+``` 
+
+Here it's an example on how to call this function:
+
+```js
+import { Buffer } from "buffer";
+import { Accounts } from 'meteor/accounts-base';
+
+--
+  
+const [qrCode, setQrCode] = useState(null);
+
+--
+  
+<button
+  onClick={() => {
+    Accounts.generate2faActivationQrCode((err, svg) => {
+      if (err) {console.error("...", err);return;}
+      /*
+        the svg can be converted to base64, then be used like: 
+         <img 
+            width="200"
+            src={`data:image/svg+xml;base64,${qrCode}`}
+         />
+      */
+      setQrCode(Buffer.from(svg).toString('base64'));
+    })
+  }}
+>
+  Generate a new code
+</button>
+```
+
+
+At this point, the 2FA won't be activated just yet. Now that the user has access to the codes generated by their authenticator app, you can call the function `Accounts.enableUser2fa`:
+
+{% apibox "Accounts.enableUser2fa" "module":"accounts-base" %}
+
+Called with a code the user will receive from the authenticator app once they read the QR code. This function throws an error on failure. If the code provided is correct, a `type` will be added to the user's `twoFactorAuthentication` object, and now the 2FA will be considered enabled:
+
+```js
+twoFactorAuthetication: {
+  type: "otp",
+  secret: "***",
+}
+```
+
+<h3 id="log-in-with-2fa">Log in with 2FA</h3>
+
+Now that you have a way to allow your users to enable 2FA on their accounts, you can create a login flow based on that.
+
+To verify if a user has or not 2FA enabled you can call the function `Accounts.has2FAEnabled`:
+
+{% apibox "Accounts.has2faEnabled" "module":"accounts-base" %}
+
+With this function, you can verify if the has or not 2FA enabled, and based on this information, you can directly log the user in the 2FA is not enabled, or redirect the user to a place where they can provide a code, in case they do have 2FA enabled.
```

Comment:
Suggestion:

```suggestion
With this function, you can check whether or not the user has 2FA enabled, and based on this information, you can directly log the user in the 2FA is not enabled, or redirect the user to a place where they can provide a code, in case they do have 2FA enabled.
```

I didn't get what you mean here:

> you can directly log the user in the 2FA is not enabled

Can you try to rephrase it?